src/Microsoft.AspNet.Mvc.Abstractions/project.json

```diff
@@ -25,7 +25,8 @@
     "Microsoft.Extensions.PropertyHelper.Sources": {
       "version": "1.0.0-*",
       "type": "build"
-    }
+    },
+    "Microsoft.Net.Http.Headers": "1.0.0-*"
```

Comment:
Ignore this change from the first commit. Did this only to ensure build picks up the local changes to `HttpAbstractions`.